leasing/models/invoice.py

```diff
@@ -1,11 +1,23 @@
+from xml.etree import ElementTree
+
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
 from enumfields import EnumField
 
 from leasing.enums import InvoiceState
+from leasing.models import Contact
 from leasing.models.mixins import TimestampedModelMixin
 
 
+def get_laske_value(name):
+    from django.conf import settings
+
+    if not hasattr(settings, 'LASKE_VALUES'):
+        return ''
+
+    return getattr(settings, 'LASKE_VALUES').get(name, '')
```

Comment:
You may simplify this hasattr+getattr to single `return getattr(settings, 'LASKE_VALUES', {}).get(name, '')`